### PR TITLE
Updated Web Components to v0.13.1

### LINKED ISSUE #36: Update to Web Components v0.13.1
Update our UI5 Web Components dependency to the latest released version.

### PR DESCRIPTION
**Pull request by MarcusNotheis**
## Features
**Card**: Added `headerInteractive` prop
**StandardListItem**: Added `info` and `infoState` props

## Breaking Changes
**Select**: the parameter of the change event is now called `selectedOption`
**Select**: Use `Option` instead of `StandardListItem` for selection
**Link**: Use design with `LinkDesign`,  type with `LinkType` is deleted
**Button**: Use design with `ButtonDesign`, type with `ButtonDesign` is deleted
**ToggleButton**: Use design with `ButtonDesign`, type with `ButtonDesign` is deleted
**Switch**: type is removed, use `graphical`
**MessageStrip**: `hideIcon` and `hideCloseButton` are renamed to `noIcon` and `noCloseButton`
**Popover**: `hideArrow` and `hideHeader` are renamed to `noArrow` and `noHeader`

Please refer to the [UI5 Web Components Changelog](https://github.com/SAP/ui5-webcomponents/releases/tag/v0.13.1) as well.

Closes #36

### CODE REVIEW

File: packages/fiori3/src/webComponents/Card/index.tsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ export interface CardPropTypes extends WithWebComponentPropTypes {
   heading?: string; // @generated
   subtitle?: string; // @generated
   status?: string; // @generated
+  headerInteractive?: boolean; // @generated
```
Comment (vbersch) [inline packages/fiori3/src/webComponents/Card/index.tsx]: We could add headerInteractive property to Card Demo. But could also be separate PR because Card demo looks kind of messy in general.

Comment (MarcusNotheis) [reply packages/fiori3/src/webComponents/Card/index.tsx]: is fixed
